.travis.yml

```diff
@@ -29,31 +39,38 @@ before_install:
       resource_group="${test_name}$(date +%s)";
       echo Run playbook $file;
 
-      travis_wait 50 ansible-playbook $file --extra-vars "{\"azure_subscription_id\":$AZURE_SUBSCRIPTION_ID,\"azure_tenant\":$AZURE_TENANT,\"azure_client_id\":$AZURE_CLIENT_ID}";
+      travis_wait 50 ansible-playbook $file --extra-vars "{\"azure_subscription_id\":$AZURE_SUBSCRIPTION_ID,\"azure_tenant\":$AZURE_TENANT,\"azure_client_id\":$AZURE_CLIENT_ID,\"resource_group_name\":$resource_group}";
 
       if [ $? -ne 0 ]; then
         echo test $file failed;
         message+=($file);
         export exit_code=1;
       fi;
-      travis_wait 50 delete_resource_group $resource_group;
+
+      travis_wait 50 ansible-playbook ./__helpers/__delete_test_resource_group.yml --extra-vars "{\"azure_subscription_id\":$AZURE_SUBSCRIPTION_ID,\"azure_tenant\":$AZURE_TENANT,\"azure_client_id\":$AZURE_CLIENT_ID,\"resource_group_name\":$resource_group}";
     }
 
 
   - scan_test() {
-      end=$((id * number_per_job));
-      start=$((end - number_per_job));
-
-      liststr=./*.yml;
-      list=($liststr);
-
-      echo start = $start, end = $end, list_lenth = ${#list[@]};
+        if [ $id == "devel-1" ]; then
+          run_test mysql_create.yml;
```

Comment:
how about define some lists for easy maintainance in future.

Reply:
define lists for playbooks groups that should be run against different ansible version.

Reply:
i want to do it later when we have all the tests running. right now i want to keep it simple.
in the future i want 3rd party contributors to be able to add tests easily.

Reply:
this is code issue, i think. not something depends on ansible bug fixing. it should be fixed in this pr.

Reply:
no i don't agree. i think this solution is good enough right now.
it's not a bug, it can be just called improvement.
the priority is to:
- make all the tests running
- make it more convenient to 3rd party developers.